tests/cases/BankAccountsTest.php

```diff
@@ -1,20 +1,23 @@
 <?php
+
 namespace MangoPay\Tests;
 require_once 'base.php';
```

Comment:
The `require` statements itself should be dropped entirely. Instead, add an `autoload-dev` directive in composer.json and use the autoload file as bootstrap file in the PHPUnit configuration. If the tests require a bootstrap file including logic, simply `require` the autoloader there.